templates/puppet.spec

```diff
@@ -0,0 +1,52 @@
+%{!?upstream_version: %global upstream_version %{version}%{?milestone}}
+{% if metadata.from_puppetlabs -%}
+%%define upstream_name {{ metadata.puppetlabs_name }}
+{% endif %}
+Name:                   {{ metadata.name }}
+Version:                XXX
+Release:                XXX
+Summary:                {{ metadata.summary }}
+License:                {{ metadata.license }}
+
+URL:                    {{ metadata.project_page }}
+
+Source0:                {{ metadata.source0 }}
```

Comment:
Not required here but just a thought, perhaps have Source be a list and then iterate over it and name them accordingly (Source0, Source1, etc.).. a bit like the metadata.dependencies.